src/nvim/options.lua

```diff
@@ -812,7 +812,7 @@ return {
       vi_def=true,
       redraw={'all_windows'},
       varname='p_fcs',
-      defaults={if_true={vi="vert:|,fold:-"}}
+      defaults={if_true={vi=""}}
```

Comment:
I see now why this was necessary. Also added a test in #8035

Reply:
sry I wanted to test again to make sure I was not inducing you into errors but had this screen pb and not enough time to fix it. I am glad you figured it out it's a cool improvement